Sources/TimelaneCombine/TimelaneCombine.swift

```diff
@@ -45,9 +50,11 @@ extension Publishers {
                     
                     subscriber.receive(subscription: sub)
                 },
-                receiveValue: { value -> Subscribers.Demand in
+                receiveValue: { [weak self] value -> Subscribers.Demand in
+                    let transform = self?.transformValue ?? { String(describing: $0) }
```

Comment:
`transformValue` is a non optional - either capture `transformValue` inside the method or unwrap `self` please